### PR TITLE
built clone

### PR DESCRIPTION
**Pull request by edwinmoradian90**

### CODE REVIEW

**Comment by stratospherique** [comment]
### Pull request under the review by Ahmed Mahfoudh (TSE)

**Comment by stratospherique** [comment]
> ### Pull request under the review by Ahmed Mahfoudh (TSE)
Good effort :clap: . However, you need to consider implementing those requirements in order to get the approval to merge to master: 

## #navigation bar
* [ ] The leftmost part needs to contain links (not just images and texts) that are vertically centered.
* [ ] It has the NY Times image logo as a link embedded in the center.

## #main content (#main body)
* [ ] Your paragraphs are centered using `text-align: center;`, try to use other positioning and float techniques to center them.

## #editors' Picks
* [ ] add an "editors pick" sections using `<aside></aside>` and align it to the right (i.e. next to your main content) using floats.

## #bottom part
* [ ] Bottom part is not your footer, take it outside of your <footer> element.
* [ ] It contains links to other articles with image previews on the leftmost side.
* [ ] It contains links to more popular articles on the rightmost side.

## #footer

* [ ] It should have a fixed position (it sticks to the bottom part of the screen).

## #important notes

* [ ] Your elements should be aligned with [floats](https://css-tricks.com/all-about-floats/)
* [ ] Since you have committed you project at once. I advice you to use  use Github/Git Flow and commit your changes often.
* [ ] Run you project through [W3C validator](https://validator.w3.org) and fix the errors.

Please consider implementing this requirement ASAP. I know for sure that you're gonna pull this through. :muscle: :muscle: 
.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+.main-icons {
+    height: 30px;
+    width: 30px;
+    border-radius: 50%;
+    background-color: purple;
+    display: inline-block;
+}
+
+#main-body {
+    text-align: center;
```
Comment (stratospherique) [inline style.css]: Use floats to align elements

**Comment by edwinmoradian90** [comment]
Oh I forgot at project requirements :/

So then I should make a small container and all the floats will stack since the container is very small?

**Comment by edwinmoradian90** [comment]
Ok cool, also does a pull request consist of the sum of your pushes or just the final push?

**Comment by stratospherique** [comment]
> Ok cool, also does a pull request consist of the sum of your pushes or just the final push?

You should follow the [Github/Git Flow](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/)  and commit your changes often. [OPTIONAL] You can commit each time you complete a requirement.

**Comment by edwinmoradian90** [comment]
So completely starting over from scratch. Also, besides float, what other position techniques can I use?

**Comment by stratospherique** [comment]
> So completely starting over from scratch. Also, besides float, what other position techniques can I use?

For this project, it's better to use floats in order to understand how they work. :wink:

**Comment by edwinmoradian90** [comment]
How do I know once you've accepted this last push I've made so I merge it to master?

**Comment by stratospherique** [comment]
> How do I know once you've accepted this last push I've made so I merge it to master?

You need to ask for a code review again and if no further changes are required then you can mege and submit the project completion form. But first, you have to finish implementing all the requirements, :wink:

**Comment by edwinmoradian90** [comment]
Cool thanks!

**Pull request merged**